src/functions/template-tags/general.php

```diff
@@ -1135,7 +1135,7 @@ function tribe_get_days_between( $start_date, $end_date, $day_cutoff = '00:00' )
 			$end_date->modify( '-1 day' );
 		}
 
-		return Tribe__Events__Date_Utils::dateDiff( $start_date->format( 'Y-m-d ' . $day_cutoff ), $end_date->format( 'Y-m-d ' . $day_cutoff ) );
+		return Tribe__Events__Date_Utils::date_diff( $start_date->format( 'Y-m-d ' . $day_cutoff ), $end_date->format( 'Y-m-d ' . $day_cutoff ) );
```

Comment:
I see that you've changed this method's name, but you didn't change on `Tribe__Events__Date_Utils`;

My question is, will you do each file in a Pull Request or do everything here then merge?

If you are going to do one class by PR, this change is not meant to be here right?